stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/server-config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
There is no single chart that has differently,  could test it locally if you get desired behaviour after changing it to `checksum/config `